src/EnergyPlus/SolarShading.cc

```diff
@@ -3639,17 +3636,26 @@ namespace SolarShading {
 		int SurfNum; // Surface Loop index
 		Real64 Fac1WoShdg; // Intermediate calculation factor, without shading
 		Real64 Fac1WithShdg; // Intermediate calculation factor, with shading
+		int IPhi; // Altitude step counter
+		int ITheta; // Azimuth step counter
+		Real64 DPhi; // Altitude step size
+		Real64 DTheta; // Azimuth step size
+		Real64 DThetaDPhi; // Product of DTheta and DPhi
+		Real64 PhiMin; // Minimum altitude
+		Real64 Phi; // Altitude angle
+		Real64 Theta; // Azimuth angle
+		Real64 FracIlluminated; // Fraction of surface area illuminated by a sky patch
```

Comment:
Can these variables be defined in the little block below when they are used?  I'm commenting as I read, so I may answer my own question shortly.